### DISCUSSION TITLE
Jigsaw Puzzle pieces from an image (PAID Freelance)

### DISCUSSION THREAD

**Discussion by Sebastianam** [pixijs/pixijs#8659] [Jobs]
Generate jigsaw-puzzle-pieces from an image, for use in a game,
- Custom number of pieces (4x4,8x8,10x10 etc.)
- With and without edge highlight and shading (for a depth effect)
- Each piece must be a small texture of its own, not just the large image masked off with Graphics, but optimised for performance.

Could probably be done using Texture.fromCanvas and making a mask using:
context.beginPath();
context.arc();
context.fillStyle = '#000000';
context.fill();
context.beginPath();
context.moveTo();
context.lineTo();
context.stroke();
context.bezierCurveTo()
context.arcTo()

Any other ways are also welcome.

Name your price! :)

**Comment by huxinfeng** [comment]
10000$
